src/app/features/home/post-capture-tab/post-capture-tab.component.ts

```diff
@@ -30,6 +30,7 @@ export class PostCaptureTabComponent implements OnInit {
     this._focus$.next(focus);
   }
 
+  categories!: 'Photo';
```

Comment:
It does not define the `categories` variable. It is just a declaration with type, which makes the app not rendering the photo page at the beginning.